packages/kbn-discover-utils/src/components/custom_control_columns/degraded_docs_control.tsx

```diff
@@ -119,3 +129,48 @@ const DegradedDocs = ({
     />
   );
 };
+
+const EnableESQLDegradedDocsControl = ({
+  addIgnoredMetadataToQuery,
+  Control,
+}: {
+  addIgnoredMetadataToQuery: DegradedDocsControlProps['addIgnoredMetadataToQuery'];
+  Control: RowControlComponent;
+}) => {
+  const [isPopoverOpen, { off: closePopover, toggle: togglePopover }] = useBoolean(false);
+
+  return (
+    <EuiPopover
+      anchorPosition="upCenter"
+      button={
+        <Control
+          css={{ color: euiThemeVars.euiColorDisabledText }}
```

Comment:
Can we use `color` prop instead of adding `css`?

Reply:
The EuiButtonIcon `color` property only accepts `"text" | "accent" | "primary" | "success" | "warning" | "danger"` colors, which does not include the `disabled` color option. I don't like this solution as well, but it was the only workaround I could quickly come up with given the current interface.